int2lm/TAG

```diff
@@ -0,0 +1 @@
+c2sm/nvhpc:21.3-runtime-cuda11.2-ubuntu20.04-int2lm
```

Comment:
The free-plan on Dockerhub only gives us 1 private repo.
For now, at develop-stage I propose to use this tag instead:
```
c2sm/cosmo:int2lm
```
We then can store both cosmo and int2lm in the same private repo